bundle/manifests/servicemeshoperator3.clusterserviceversion.yaml

```diff
@@ -496,10 +496,10 @@ spec:
           template:
             metadata:
               annotations:
-                images.v1_21_2.cni: registry.redhat.io/openshift-service-mesh/istio-cni-rhel9:1.21.2.tp.1
-                images.v1_21_2.istiod: registry.redhat.io/openshift-service-mesh/istio-pilot-rhel9:1.21.2.tp.1
-                images.v1_21_2.proxy: registry.redhat.io/openshift-service-mesh/istio-proxyv2-rhel9:3.0.0.tp.1
-                images.v1_21_2.ztunnel: registry.redhat.io/openshift-service-mesh/istio-pilot-rhel9:1.21.2.tp.1
+                images.v1_22_1.cni: registry.redhat.io/openshift-service-mesh/istio-cni-rhel9:1.22.1.tp.1
```

Comment:
Question: Should we add the new versions instead of replacing them?

Reply:
There will be only one supported version for TP1, so I think we need to replace this. We have to change this after TP1

Reply:
Should we override `versions.yaml` file so that it only contains the one supported version then?

Reply:
ah ok, I thought this PR was against that branch, not main...

Reply:
Yeah, this change will be more relevant for `release-3.0-tp1` but we want to stay in sync.